pkg/liquidity_provider.go

```diff
@@ -84,6 +86,67 @@ type CredentialsUpdateRequest struct {
 	NewPassword string `json:"newPassword" validate:"required"`
 }
 
+type GetReportsPeginPegoutRequest struct {
+	StartDate string `json:"startDate" validate:"required,datetime=2006-01-02"`
+	EndDate   string `json:"endDate" validate:"required,datetime=2006-01-02"`
+}
+
+func (r *GetReportsPeginPegoutRequest) ValidateGetReportsPeginPegoutRequest() error {
+	if r.StartDate == "" {
+		return errors.New("startDate is required")
+	}
+	if r.EndDate == "" {
+		return errors.New("endDate is required")
+	}
+
+	const dateFormat = "2006-01-02"
```

Comment:
This string is used internally by the std lib there is no need to hardcode it, the std lib already has the constants for this...